### DISCUSSION TITLE
Similarities and differences between SimPhoNy and approaches of the Semantic Web community

### DISCUSSION THREAD

**Discussion by pablo-de-andres** [simphony/simphony-osp#689] [General]
In GitLab by @urbanmatthias on Jun 17, 2020, 15:02

This issue is meant to collect similarities and differences between SimPhoNy and approaches of the Semantic Web community.
@yoavnash @abahde please comment if you want to add anything to this list.

**Comment by pablo-de-andres** [comment]
In GitLab by @urbanmatthias on Jun 17, 2020, 15:03

changed title from **Similarities and difference between SimPhoNy and approaches of the Semantic Web community** to **Similarities and difference{+s+} between SimPhoNy and approaches of the Semantic Web community**

**Comment by pablo-de-andres** [comment]
In GitLab by @yoavnash on Jun 17, 2020, 17:17


#### Used by the Semantic Web community members but not supported/used in SimPhoNy:
1. DCAT: 
   - Would nice to demonstrate how it will look like in OSP-core
   - Should there be additional support for handling the dataset in OSP-core?
1. SPARQL:
   - A standard query language that is currently not supported.
   - An idea that allows us to support it: redesign the DB schema of the SQL wrapper so that everything is stored in a triple store. This way, SPARQL queries can be run directly in that DB as it is done in OwlReady.
1. Reasoning:
   - Currently not supported 
 
***

#### Innovation in SimPhoNy compared to common practices in the Semantic Web community:
1. Conversion mechanism of data originating from various software tools and direct interaction with them.
1. CUDS API for an ontology-based and uniform representation of data.
1. A management system for *dataspaces* (that is, a virtual space that accommodates transparent data).
1. Semantic data validation that goes beyond what Excel offers (by checking data against a data model that is ontology-based).

**Comment by yoavnash** [comment]
May be relevant: https://sparqlwrapper.readthedocs.io/en/latest/index.html
